crash_alert/README.md

```diff
@@ -0,0 +1,48 @@
+# Crash Alert
+
+*This directory contains scripts to listen to the crash detection system and send alerts to subscribed users' devices when the bike crashes*
+
+
+- [Crash Alert](#crash-alert)
+  - [About](#about)
+  - [Basic Setup and Usage](#basic-setup-and-usage)
+
+## About
+Upon receiving a true from the crash detection MQTT channel, the program sends an alert via this Slack channel in the MHP Slack workspace. This can be extended for more APIs by adding children of the MessageSender class and implementing the send_message() method with the APIs respective calls.
+
+## Basic Setup and Usage
+These steps assume that you have [poetry](https://python-poetry.org/) installed.
+
+```bash
+# Move into crash_alert directory
+cd crash_alert
+
+# Create a poetry environment using Python 3.7
+poetry env use 3.7
+
+# Enter the environment
+poetry shell
+
+# Install dependencies
+poetry install
+```
+
```

Comment:
Add some instructions on setting up the `.env` file, something that tells them where to get the _Slack Webhook from_. Otherwise, the README is fine.